projects/client-child-process/connector.ts

```diff
@@ -0,0 +1,51 @@
+import { ClientConnector, connectClient, applyApi, Client } from '@remixproject/plugin/connector'
+import { PluginClient } from '@remixproject/plugin/client'
+import { Message } from '../utils/src/types/message'
+import { RemixApi } from '../utils/src/api/remix-profile'
+import { Api, ApiMap } from '../utils/src/types/api'
+
+
+export const childProcessConnector: ClientConnector = {
+
+  /** Send a message to the engine */
+  send(message: Partial<Message>) {
+    process.send(message)
+    this.websocket.send(JSON.stringify(message))
+  },
+
+  /** Get messae from the engine */
+  on(cb: (message: Partial<Message>) => void) {
+    process.on('message', (event) => cb(event))
+  }
+}
+
+/**
+ * Connect an Iframe client to a web engine
```

Comment:
you mean `Connect a child process client to a web engine` right?

Reply:
indeed